cmd/frontend/internal/app/pkg/updatecheck/client.go

```diff
@@ -142,6 +147,18 @@ func authProviderTypes() []string {
 	return types
 }
 
+func externalServiceKinds() ([]string, error) {
+	services, err := db.ExternalServices.List(context.Background(), db.ExternalServicesListOptions{Kinds: []string{}})
```

Comment:
The default value for a slice has a zero length. So this can just be:

```suggestion
	services, err := db.ExternalServices.List(context.Background(), db.ExternalServicesListOptions{})
```